src/storage-types/src/sources.rs

```diff
@@ -68,6 +71,38 @@ pub use crate::sources::postgres::PostgresSourceConnection;
 
 include!(concat!(env!("OUT_DIR"), "/mz_storage_types.sources.rs"));
 
+/// A fraternal twin of [`UnresolvedItemName`] that can implement [`Arbitrary`].
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct ExportReference(pub Vec<Ident>);
+
+impl proptest::prelude::Arbitrary for ExportReference {
+    type Parameters = ();
+    type Strategy = BoxedStrategy<Self>;
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
+        // Generate a set of valid `Ident` strings.
+        let string_strategy = string_regex("^[a-zA-Z_][a-zA-Z0-9_$]{3,10}$").unwrap();
+
+        vec(string_strategy, 0..100)
+            .prop_map(|inner| {
+                ExportReference(inner.into_iter().map(|i| Ident::new(i).unwrap()).collect())
+            })
+            .boxed()
+    }
+}
+
+impl std::fmt::Display for ExportReference {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", itertools::join(&self.0, "."))
+    }
+}
```

Comment:
Do we use this anywhere? If not, I'd love to get rid of it, as its a compatibility hazard if `UnresolvedItemName`'s `Display` impl doesn't match exactly

Reply:
Oh yeah––good call. ty.